src/shogun/io/SGIO.cpp

```diff
@@ -189,6 +196,7 @@ void SGIO::progress(
 			if (current_val >= max_val-1)
 			{
 				v = 100;
+				last_progress=v-1e-6;
```

Comment:
Hehe ;)
We gotta make everything as good as we possibly can -- chances that somebody touches code again are pretty low, so better get it correct (and neat) in the first place.

Hey are you in IRC at some point, I remember you asked about dependencies ....